backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/OrganizationRole.java

```diff
@@ -1,16 +1,30 @@
 package gov.cdc.usds.simplereport.config.authorization;
 
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.Set;
+
 public enum OrganizationRole {
-    USER("Users"), 
-    ADMIN("Admins"); // ?
+    ENTRY_ONLY("Test-entry users",
+            EnumSet.of(UserPermission.START_TEST, UserPermission.UPDATE_TEST, UserPermission.SUBMIT_TEST)),
+    USER("Users",
+            EnumSet.of(UserPermission.READ_PATIENT_LIST, UserPermission.READ_RESULT_LIST, UserPermission.EDIT_PATIENT,
+                    UserPermission.START_TEST, UserPermission.UPDATE_TEST)),
```

Comment:
Out of curiosity: when is the "USER" role used? 

The specs (confusingly spread across two issues):
https://github.com/CDCgov/prime-simplereport/issues/642
https://app.zenhub.com/workspaces/simplereport-5ff8be8854ae8b000e378210/issues/cdcgov/prime-simplereport/570

Based on the above, the two user types are ENTRY_ONLY and ADMIN (distinct from superadmins, which I am ignoring)

Sounds like

Reply:
The spec actually extends back to issue #405!

Roughly speaking, right now USER is the role that everybody has, which is referred to in various conversations as "Basic User" or "Standard User". Admin users will get permissions on top of those; entry-only users will have permissions subtracted, which is a _lousy_ way to set up your RBAC model but here we are.

Reply:
thanks for the clarification! Lots of naming confusions